package.json

```diff
@@ -119,6 +119,7 @@
     "lodash": "^4.17.21",
     "memoizee": "^0.4.15",
     "mermaid": "^8.13.9",
+    "moment": "^2.29.3",
```

Comment:
It would be nice if we could just rely on `date-fns` which we already use. Ideally the SDK & UI should move away from momentjs to something lighter...

Reply:
Captured it here: https://github.com/balena-io-modules/rendition/issues/1608